### PR TITLE
docs: add deploy to Zeabur

### PR DESCRIPTION
**Pull request by MichaelYuhe**
#### Description (required)

Add deploy to Zeabur guide for en and zh-cn as another deploy option.

#### Related issues & labels (optional)

- Suggested label: documentation

### CODE REVIEW

**Review by dreyfus92 — COMMENTED** [review]
Thank you for adding this guide @MichaelYuhe 😁, just a comment while I let our docs lead review this PR, we count with a i18n tracker (Lunaria), which will let our amazing translators to handle the translation in their respective language once this is merged. So I'd recommend you to just leave the english version of it. I might as well make a review as soon as I can.

**Comment by MichaelYuhe** [comment]
Thanks! I will remove the zh-cn file.

File: src/content/docs/en/guides/deploy/zeabur.mdx
Comment (dreyfus92) [inline src/content/docs/en/guides/deploy/zeabur.mdx:5]: I think that this should be set to true.
```suggestion
i18nReady: true
```

Comment (MichaelYuhe) [reply src/content/docs/en/guides/deploy/zeabur.mdx]: Thanks! Sorry that I thought it should be false until every language is finished :D

File: src/content/docs/en/guides/deploy/zeabur.mdx
Comment (VoxelMC) [inline src/content/docs/en/guides/deploy/zeabur.mdx:7]:
```suggestion
[Zeabur](https://zeabur.com) offers hosting for full-stack web applications. Any Astro site can be hosted on Zeabur as an SSR or static site.
```

File: src/content/docs/en/guides/deploy/zeabur.mdx
Comment (VoxelMC) [inline src/content/docs/en/guides/deploy/zeabur.mdx:21]: I would suggest adding a step before `1.` about adding the `@astrojs/node` SSR adapter.

Have a look the [AWS Flightcontrol](https://docs-git-fork-michaelyuhe-main-astrodotbuild.vercel.app/en/guides/deploy/flightcontrol/) deployment recipe to see what I mean :)

File: src/content/docs/en/guides/deploy/zeabur.mdx
Comment (VoxelMC) [inline src/content/docs/en/guides/deploy/zeabur.mdx:42]:
```suggestion
    ```json
    {
      "scripts": {
        "start": "node dist/server/entry.mjs"
      }
    }
    ```
```

Indent these code blocks to maintain consistency across the docs :)

File: src/content/docs/en/guides/deploy/zeabur.mdx
Comment (VoxelMC) [inline src/content/docs/en/guides/deploy/zeabur.mdx:32]:
```suggestion
    ```js
    import node from '@astrojs/node';
    
    export default defineConfig({
      output: 'server',
      adapter: node({
        mode: "standalone"
      })
    })
    ```
```

**Review by VoxelMC — COMMENTED** [review]
This looks great!!! Very clear and concise - I love it. Thank you for your hard work on this, I can't wait to try it out someday :)

I left a couple of suggestions and some formatting changes, I hope they help!!

File: src/content/docs/en/guides/deploy/zeabur.mdx
Comment (MichaelYuhe) [reply src/content/docs/en/guides/deploy/zeabur.mdx:21]: Thank you, I will add the step :)

**Comment by MichaelYuhe** [comment]
> This looks great!!! Very clear and concise - I love it. Thank you for your hard work on this, I can't wait to try it out someday :)
> 
> I left a couple of suggestions and some formatting changes, I hope they help!!

Thanks, I've updated and applied the suggestions!

File: src/content/docs/en/guides/deploy/zeabur.mdx
Comment (ElianCodes) [inline src/content/docs/en/guides/deploy/zeabur.mdx:7]:
```suggestion
[Zeabur](https://zeabur.com) offers hosting for full-stack web applications. Astro sites can be hosted as both SSR or static output.
```

File: src/content/docs/en/guides/deploy/zeabur.mdx
Comment (ElianCodes) [inline src/content/docs/en/guides/deploy/zeabur.mdx:13]: I think we can drop this sentence, since it's basically a repetition of the first sentence in the guide.

File: src/content/docs/en/guides/deploy/zeabur.mdx
Comment (ElianCodes) [inline src/content/docs/en/guides/deploy/zeabur.mdx:17]:
```suggestion
Astro outputs a static site by default. There is no need for any extra configuration to deploy a static Astro site to Zeabur. 
```

File: src/content/docs/en/guides/deploy/zeabur.mdx
Comment (ElianCodes) [inline src/content/docs/en/guides/deploy/zeabur.mdx:21]:
```suggestion
To deploy with SSR support, set up the @astrojs/node adapter first. Then, follow the next steps to configure your Astro site for SSR:
```

File: src/content/docs/en/guides/deploy/zeabur.mdx
Comment (ElianCodes) [inline src/content/docs/en/guides/deploy/zeabur.mdx:34]:
``````suggestion
    ```js title="astro.config.mjs"
    import node from '@astrojs/node';
    
    export default defineConfig({
      output: 'server',
      adapter: node({
        mode: "standalone"
      })
    })
    ```
``````

File: src/content/docs/en/guides/deploy/zeabur.mdx
Comment (ElianCodes) [inline src/content/docs/en/guides/deploy/zeabur.mdx:44]:
``````suggestion
    ```json title="package.json"
    {
      "scripts": {
        "start": "node dist/server/entry.mjs"
      }
    }
    ```
``````

File: src/content/docs/en/guides/deploy/zeabur.mdx
Comment (ElianCodes) [inline src/content/docs/en/guides/deploy/zeabur.mdx:50]: I would opt to just remove both sentences here, since they don't add any extra value
```suggestion
```

**Review by ElianCodes — COMMENTED** [review]
hey there! Thank you so much for the PR!

Here's the first review, I added some suggestions. After this,  I leave it to @sarah11918 to final review and merge!

File: src/content/docs/en/guides/deploy/zeabur.mdx
Comment (MichaelYuhe) [reply src/content/docs/en/guides/deploy/zeabur.mdx:50]: That's because Zeabur's CLI is still in a beta version, I'll remove these and come back with another pr when the CLI is ready!

**Comment by MichaelYuhe** [comment]
Thanks for the review! I've updated the documentation to apply the changes

**Comment by ElianCodes** [comment]
Sounds good! Thanks

File: src/content/docs/en/guides/deploy/zeabur.mdx
Comment (sarah11918) [inline src/content/docs/en/guides/deploy/zeabur.mdx:36]:
```suggestion
    ```json title="package.json" {3}
```
We can get fancy and highlight the line!  :nail_care:

File: src/content/docs/en/guides/deploy/zeabur.mdx
Comment (sarah11918) [inline src/content/docs/en/guides/deploy/zeabur.mdx:48]:
```suggestion
1. Click <kbd>Create new project</kbd> in the [Zeabur dashboard](https://dash.zeabur.com).
```
missing period at the end of the sentence

**Review by sarah11918 — COMMENTED** [review]
This looks great, @MichaelYuhe ! What a great first contribution!  :tada: 

Only the tiniest of changes, plus I do think we can get away with linking to the Node adapter installation instructions, for the reasons I mentioned. What do you think?

File: src/content/docs/en/guides/deploy/zeabur.mdx
Comment (MichaelYuhe) [reply src/content/docs/en/guides/deploy/zeabur.mdx:36]: Magic number!

File: src/content/docs/en/guides/deploy/zeabur.mdx
Comment (MichaelYuhe) [reply src/content/docs/en/guides/deploy/zeabur.mdx:32]: Great suggestion! My bad for not reading other docs carefully :(

**Comment by MichaelYuhe** [comment]
> This looks great, @MichaelYuhe ! What a great first contribution! 🎉
> 
> Only the tiniest of changes, plus I do think we can get away with linking to the Node adapter installation instructions, for the reasons I mentioned. What do you think?

Great suggestion! Applied

**Review by sarah11918 — APPROVED** [review]
This looks fantastic, @MichaelYuhe ! So happy to have this new contribution. Welcome to Team Docs!  :partying_face:

**Pull request merged**
